README.md

```diff
@@ -379,17 +411,101 @@ Documentation Forthcoming
+
+###### `RenderModule`
+<!-- ONE-MARKDOWN:START url="https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md" selector="### `RenderModule`" -->
+Please see [`RenderModule`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#rendermodule) in the Holocron API.
+<!-- ONE-MARKDOWN:END -->
+
+###### `holocronModule`
+
+<!-- ONE-MARKDOWN:START url="https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md" selector="### `holocronModule`" -->
+Please see [`holocronModule`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#holocronmodule) in the Holocron API.
+<!-- ONE-MARKDOWN:END -->
+
+###### `composeModules`
+
+<!-- ONE-MARKDOWN:START url="https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md" selector="### Compose Modules" -->
+Please see [`composeModules`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#composemodules) in the Holocron API.
+<!-- ONE-MARKDOWN:END -->
+
+#### Loading Data
+
+When [Holocron Modules](#modules) are composed and loaded on the Server and Client, the `loadModuleData` Module Lifecycle Hook is called to load any async requests.
+
+**Contents**
+* [`loadModuleData`](#loadmoduledata)
+
+##### `loadModuleData`
+
+**Runs On**
+* ✅ Server
+* ✅ Browser
+
+**Shape**
+```js
+HelloWorldModule.loadModuleData = async ({ store, fetchClient }) => {};
+```
+
+**Arguments**
+
+| Argument | Type     | Description                     |
+|----------|----------|---------------------------------|
+| `store`   | [`Redux Store`](https://redux.js.org/api/store/) | Redux store containing `getState`, `dispatch` and [other methods](https://redux.js.org/api/store/). |
+| `fetchClient`   | [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) | [ES6 Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) Compatible Client. |
+
+The `loadModuleData` Module Lifecycle Hook, is executed on the 1) Server and 2) Browser when a Module is loaded in either environment. This method is executed and resolved before any React Components are rendered inside a Holocron Module.
+
+In practice, we may [`dispatch`](https://redux.js.org/api/store/#dispatchaction) Redux actions and make [`async/await`](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await) requests to populate our Module's reducers before any React Components are rendered:
+
+```js
+// Runs on both Server and Browser
+HelloWorldModule.loadModuleData = async ({ store, fetchClient }) => {
```

Comment:
Can we link to where you can customize the `fetchClient`. I think we have that documented already right?